my-app/src/redux/useCasesSlice.js

```diff
@@ -12,7 +12,32 @@ export const getUseCases  = createAsyncThunk(
 export const useCasesSlice = createSlice({
     name: 'useCases',
     initialState: {
-      useCases: [],
+      useCases: [{
+        "_id": "5e80a67fa39b65001fafd0f4",
+        "name": "New in",
+        "slug": "new-in",
+        "behavior": "NEW_ARRIVAL",
+        "description": "Stunning product video ads, automatically created based on products you uploaded to your store in the past 30 days. We've placed your product images in pre-built, proven templates so you can increase sales without any effort.",
+        "isEnabled": true,
+        "createdAt": "2020-03-29T13:45:35.402Z",
+        "updatedAt": "2020-07-14T07:07:07.260Z",
+        "__v": 0,
+        "order": 0,
+        "campaignId": "606d6c938e429b001ed11bb6"
```

Comment:
Why do you need all this data in the state?
I think all you need is the name (for rendering the title)
and the campaignId to fetch the videos later on